### PR TITLE
ZENKO-3629: use helm3 for CI tests

### PR DESCRIPTION
**Pull request by alexanderchan-scality**
**What does this PR do, and why do we need it?**
use `helm3`

**Which issue does this PR fix?**

fixes ZENKO-3629

**Special notes for your reviewers**:

### CODE REVIEW

File: eve/workers/end2end/requirements.sh [hunk 1]
```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
 export HELM_VER=2.16.7
+export HELM3_VER=3.5.4
```
Comment (tcarmet) [inline eve/workers/end2end/requirements.sh]: Any particular reason to not fully remove helm2?

Comment (alexanderchan-scality) [reply eve/workers/end2end/requirements.sh]: will remove after confirming that the change works.

**Review by miniscruff — APPROVED** [review]

**Review by naren-rajendran — APPROVED** [review]

**Pull request merged**